client/src/containers/frontend/CollectionDetail.js

```diff
@@ -107,7 +107,11 @@ class CollectionDetailContainer extends PureComponent {
   render() {
     const project = this.props.project;
     const collection = this.props.collection;
-    if (!project && !collection) return null;
+
+    if (!project || !collection) return null;
```

Comment:
Why the condition change? Isn't it right to return null if we're missing the project _or_ the collection?